utilities/aap.py

```
@@ -0,0 +1,619 @@
+    headers: dict[str, str] = {"Content-Type": "application/json"}
+    if token:
+        headers["Authorization"] = f"Bearer {token}"
+
+    url = f"{awx_url}{endpoint}"
+    response = requests.request(
+        method=method,
+        url=url,
+        headers=headers,
+        auth=auth,
+        json=json_data,
+        verify=False,
```

Ruff (0.15.20)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)